ExportCSV.ascx.cs

```diff
@@ -194,6 +194,11 @@ void WriteData(DataTable data, DataTable fields, string folder, string fileName,
                                              ? fieldTitle + DataTableColumn.Appendix_Original
                                              : fieldTitle);
                         var value = row[valueName].AsString();
+                        if (fieldTitle == "Created at" || fieldTitle == "Changed at")
+                        {
+                            DateTime valueDate = DateTime.Parse(value);
+                            value = valueDate.ToString("dd/MM/yyyyTHH:mm:ss");
```

Comment:
This date format is really almost only US format...

Can we change this to either:
🅰️  using the thread culture
or
🅱️ an ISO date-time string

I am leaning towards B myself if people do some data processing based on it, it's more of an expected format for an export, opinions ?

Reply:
Hi, i tested this option and when use that format, time is being printed according to time zone configuration in DNN. In my case for example, if DNN is set with zone GMT -5, is showing my local time.

Anyway, can be a good idea to have an option to pick field format, next to the new option to export system fields. I can made this later if you think is a good idea too.

Reply:
Oh, I was not taking about the actual value (timezone) but the formatting. It's only in US that it's day/month/year...

I would use "yyyy-MM-ddTHH:mm:ssZ" so at least it is some documented international standard ISO 8601 and people have something standard to go by no matter where they live and their timezone too by extension since it would also carry the timezone offset in that string with the ending z format